packages/ketcher-react/src/script/editor/tool/template.ts

```diff
@@ -428,19 +430,20 @@ class TemplateTool {
       FunctionalGroup.isContractedFunctionalGroup(ci.id, functionalGroups) &&
       this.mode === 'fg'
     ) {
+      functionalGroupRemoveAction = new Action()
       const struct = this.editor.struct()
-      const closestGroup = struct.sgroups.get(ci.id)!
-      const attachmentAtomId = closestGroup.getAttAtomId(struct)
-      ci = { map: 'atoms', id: attachmentAtomId }
-
-      this.editor.update(
-        fromFragmentDeletion(this.editor.render.ctab, {
-          atoms: [...SGroup.getAtoms(struct, closestGroup)].filter(
-            (atomId) => atomId !== attachmentAtomId
-          ),
-          bonds: [...SGroup.getBonds(struct, closestGroup)]
-        })
+      const restruct = this.editor.render.ctab
+      const functionalGroupToReplace = struct.sgroups.get(ci.id)!
+      const attachmentAtomId = functionalGroupToReplace.getAttAtomId(struct)
+      const sGroupAtoms = SGroup.getAtoms(struct, functionalGroupToReplace)
+      const [_, ...atoms] = sGroupAtoms
```

Comment:
Hi, if I understand correctly, `atoms` is an array of all the S-Group atoms except for the attachment atom.
So my question is, why the first atom of `sGroupAtoms` is the attachment atom? Is it a convention?